Stripe/STPURLCallbackHandler.m

```diff
@@ -32,7 +32,7 @@ + (BOOL)handleStripeURLCallbackWithURL:(NSURL *)url FAUXPAS_IGNORED_ON_LINE(Unus
 
 @interface STPURLCallback : NSObject
 @property (nonatomic) NSURLComponents *urlComponents;
-@property (nonatomic) id<STPURLCallbackListener> listener;
+@property (nonatomic, weak) id<STPURLCallbackListener> listener;
```

Comment:
Is it potentially actually better to just leave this as it was and let the context retain itself for the duration of the redirect or until a cancel occurs? Or do you think that behavior is too unexpected.